Add min donation check

catalanojuan opened the pull request.

maraoz commented:
Tested locally and everything looks good. However, I'm suspicious about those package-lock.json changes. Will ping privately about this to investigate

catalanojuan commented:
> Tested locally, and everything looks good. However, I'm suspicious about those package-lock.json changes. Will ping privately about this to investigate

@maraoz, thanks for reviewing! I just reverted the changes on the `package-lock.json` since it runs ok anyways locally for me, and it's probably connected with us having diff npm versions. We can sort that out separately :) I also rewrote my tests using the eth to wei conversion you suggested
